recipe/meta.yaml

```diff
@@ -13,10 +13,12 @@ source:
   {{ hash_type }}: {{ hash }}
 
 build:
-  number: 0
+  number: 1
   script: 
     - python setup.py install --record=record.txt  # [not py3k]
     - python setup3.py install --record=record.txt  # [py3k]
+    - copy %PREFIX%\Lib\site-packages\pywin32_system32\*.dll %PREFIX%
```

Comment:
Not sure. I have a hunch that the `DLL`s should be in `%LIBRARY_LIB%`. @gillins, @patricksnape, or @msarahan are probably better qualified to say so.